docs/detections/alerts-view-details.asciidoc

```diff
@@ -125,31 +155,79 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
+* **Cases related to the alert**: Shows the total number and names of cases to which the alert has been added. Click a case’s name to open its details.
+
+If you have a https://www.elastic.co/pricing[Platinum or higher subscription], these details are also included:
+
+* **Alerts related by session ID**: Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the **Include session data** setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
+* **Alerts related by process ancestry**: Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click **Investigate in timeline**.
+
+[discrete]
+[[prevalence-section]]
+=== Prevalence
+
+The Prevalence sub-section shows how often data from the alert was observed on other host and user events. Prevalence calculations use data from the alert's highlighted fields and examine user and host events that occurred during the last 30 days. Click **Prevalence** to display the <<prevalence-sub-tab,Prevalence sub-tab>> in the left panel. 
+
+[discrete]
+[[prevalence-sub-tab]]
+==== Prevalence sub-tab
+
+The **Prevalence** sub-tab shows relationships with other alerts, events, users, and hosts. Update the date time picker to show data from a different time range. 
 
 [role="screenshot"]
-image::images/threat-intel-sub-sec.png[Enriched with Threat Intelligence subsection with matched file hash, 700]
+image::images/prevalence-sub-tab.png[Table within the Prevalence sub-tab, 70%]
 
-TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.
+The following details are provided:
 
-Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat Intel* tab with additional threat match data.
+* **Field**: Shows the alert's highlighted fields.
+* **Value**: Shows values from the alert's highlighted fields. 
+* **Alert count**: Shows the total number of alert documents that have identical field-value pairs, including the alert you're currently examining. 
++
+NOTE: Queries for the **Alert count** column only retrieve alert documents that have the `event.kind:signal` field-value pair.  
```

Comment:
I would remove the word `alert` _{...} in only retrieve alert documents that have the {...}_

Reply:
```suggestion
NOTE: Queries for the **Alert count** column only retrieve documents that have the `event.kind:signal` field-value pair.  
```